grants/search.js

```diff
@@ -124,32 +124,60 @@ function buildSortMeta(sort, queryParams) {
     };
 }
 
-function addActiveStatus(grant) {
+// Add an active status to currently-running projects
+function addActiveStatus({ grant, locale }) {
     const endDate = get(grant, 'plannedDates[0].endDate', false);
-
     if (endDate) {
         grant.isActive = !moment(endDate).isBefore(now);
     }
-
-    return grant;
+    return { grant, locale };
 }
 
-function addFundingProgrammeDetail(grant) {
-    const mainProgramme = head(grant.grantProgramme);
+// Try to add a URL to the funding programme where one matches
+function addFundingProgrammeDetail({ grant, locale }) {
+    const mainProgramme = grant.grantProgramme;
     if (mainProgramme) {
         const programme = get(fundingProgrammes, mainProgramme.code, false);
         if (programme && programme.urlPath) {
             mainProgramme.url = programme.urlPath;
         }
     }
-    return grant;
+    return { grant, locale };
 }
 
-function addGrantDetail(grant) {
-    return flow(
+// Translate the organisation types in the data itself as well as the facets
```

Comment:
This was spotted by @LynseyJReynolds – on the grant detail page we weren't translating these fields as only the facets were being converted.